.github/workflows/checkstyle-and-license-check.yml

```diff
@@ -18,15 +18,15 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: Check:Checkstyle and Apache Rat Check
+name: Check:Checkstyle and License Check
```

Comment:
How about rename as `Checkstyle and License Check`

Reply:
the `Check:` is a prefix, if there are more than 2 checks action, it's easy to group these action together. like followings

<img width="352" alt="image" src="https://user-images.githubusercontent.com/4112856/193736326-dceba9f1-3b63-4de7-94de-03fdeea30148.png">

Reply:
OK, then, how about rename as `Check: Checkstyle and Licenses`

Reply:
By the way, ":" should keep blank with follow chars. Please change `Docker: xxx` as well.

Reply:
it's impossible . casue in yaml file `:` with blank space will not be treated as a variable.
<img width="458" alt="image" src="https://user-images.githubusercontent.com/4112856/193737349-bd983963-88f2-452e-a35d-52d95fb930f3.png">
<img width="392" alt="image" src="https://user-images.githubusercontent.com/4112856/193737380-f632bc8a-07b7-44a5-ac8b-8bc008a170c6.png">